packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -414,13 +614,16 @@ class IOSDevice extends Device {
 
       _logger.printTrace('Application launched on the device. Waiting for Dart VM Service url.');
 
-      final int defaultTimeout = interfaceType == IOSDeviceConnectionInterface.network ? 45 : 30;
+      final int defaultTimeout = isWirelesslyConnected ? 45 : 30;
```

Comment:
Is the 30 here equivalent to `DeviceManager.minimumWirelessTimeout` or is that just coincidental?

Reply:
oh wait, no, here the 30 is if we're wired, whereas if we're wireless it's 45. I'm a little worried we're spreading out the timeout values across these files.

Reply:
@jmagman thoughts?

Reply:
Yeah so `DeviceManager.minimumWirelessTimeout` is supposed to be the time we take to discover wireless devices for device selection. The 30/45 second timeout here is how long we wait to display error message when waiting for the dart vm service to be discovered. Maybe I should rename `minimumWirelessTimeout` to `minimumWirelessDiscoveryForSelectionTimeout` or `minimumTargetDeviceWirelessDiscoveryTimeout`?